README.md

```diff
@@ -16,9 +16,8 @@ page](https://developer.arm.com/architectures/system-architectures/software-stan
 
 ## Contributing
 
-The branch ``main`` is the release branch, whose tip contains the
-latest release. Therefore, please submit your PR against the branch
-``next-release``.
+Contributions of any kind are always welcome! Please check the
+[contributions guidelines](CONTRIBUTING.md) for details.
```

Comment:
```suggestion
[contribution guidelines](CONTRIBUTING.md) for details.
```

Reply:
This actually is an example of the compound-noun plural rule that I thought was triggering with the 'specification parts' elsewhere: https://en.wikipedia.org/wiki/English_plurals#Plurals_of_compound_nouns